packages/esm-appointments-app/src/form/appointments-form.component.tsx

```diff
@@ -475,33 +484,31 @@ const AppointmentsForm: React.FC<AppointmentsFormProps> = ({
                     control={control}
                     render={({ field: { onChange, value, ref } }) => (
                       <ResponsiveWrapper>
-                        <DatePicker
-                          datePickerType="range"
-                          dateFormat={datePickerFormat}
-                          value={[value.startDate, value.recurringPatternEndDate]}
-                          ref={ref}
-                          onChange={([startDate, endDate]) => {
-                            onChange({
-                              startDate: new Date(startDate),
-                              recurringPatternEndDate: new Date(endDate),
-                              recurringPatternEndDateText: dayjs(new Date(endDate)).format(dateFormat),
-                              startDateText: dayjs(new Date(startDate)).format(dateFormat),
-                            });
-                          }}>
-                          <DatePickerInput
-                            id="startDatePickerInput"
-                            labelText={t('startDate', 'Start date')}
-                            style={{ width: '100%' }}
-                            value={watch('appointmentDateTime').startDateText}
-                          />
-                          <DatePickerInput
-                            id="endDatePickerInput"
-                            labelText={t('endDate', 'End date')}
-                            style={{ width: '100%' }}
-                            placeholder={datePickerPlaceHolder}
-                            value={watch('appointmentDateTime').recurringPatternEndDateText}
-                          />
-                        </DatePicker>
+                        <Controller
```

Comment:
I see a bunch of references to reoccurring dates in the old code, did we confirm that scheduling reoccurring appointments still works after these changes?

Reply:
Updating this value as intended doesn't cause any issues with the handling of recurring appointment end dates. Its working fine..